worldedit-bukkit/src/main/java/com/boydti/fawe/bukkit/wrapper/AsyncChunk.java

```diff
@@ -183,4 +189,9 @@ public boolean contains(@NotNull BlockData block) {
         //todo
         return false;
     }
+
+    @Override
+    public @NotNull PersistentDataContainer getPersistentDataContainer() {
+        return null;
```

Comment:
I'm not entirely sure what to do here, so I'll just return null for now

Reply:
return the value of the super Method. Any other return nulls should do the same.